src/ecs/commands/runCommandInContainer.ts

```diff
@@ -99,24 +99,10 @@ export async function runCommandInContainer(
 
     const task = taskChoice[0].label
 
-    //Get command line text from user to run in the container
-    const command = await window.showInputBox({
```

Comment:
@gsweetwood I moved this to go _after_ the "do you want to run this" prompt. Was there a reason to ask for the command _before_ showing the "do you want to run this" prompt?

Reply:
I saw it as a last chance to bail out before execution, but I personally think either works.